### ISSUE TITLE
Hello, missing file ajax.php

### ISSUE THREAD

**Issue by vudangnt** [jaimehgb/RaiWebWallet#1]
Hello, missing file ajax.php

**Comment by jaimehgb** [comment]
Yep, I've not published server side code :P 

Just what it's executed at the users' browser.

**Comment by jelofsson** [comment]
If I understand the source correctly you're sending the wallets seed to the backend?

**Comment by jaimehgb** [comment]
Nope, they are never sent to the backend. They are generated locally, ciphered with the password, and that ciphertext is sent to the server to be stored there.

**Comment by dsosby** [comment]
Any plans to support full client-side only, similar to MyEtherWallet, like having the user responsible for maintaining encrypted wallet file or a mnemonic seed?

**Comment by jaimehgb** [comment]
Hmm yeah, not a priority but definitely yes :)

**Comment by lazerscience** [comment]
@jaimehgb any plans of also open-sourcing the backend as well? I think this might be crucial if you want some security reviews by the community?

**Comment by AugustoResende** [comment]
Yes.
